tests/test_validator.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+from click.testing import CliRunner
+from pathlib import Path
+
+from validator.__main__ import cli
+
+
+class TestXMLValidator(unittest.TestCase):
+    """Test for XML validator tool.
+
+    Testing the command line functions.
+    """
+    TESTFILES_ROOT = Path(__file__).parent / 'test_files'
+
+    def setUp(self):
+        """Initialise CLI runner and set paths to test files."""
+        self.runner = CliRunner()
+
+        # XML files and Schemas as published by ENA
+        self.xml_path = self.TESTFILES_ROOT / 'xml'
+        self.xsd_path = self.TESTFILES_ROOT / 'schemas'
+
+    def tearDown(self):
+        """Remove setup variables."""
+        pass
+
+    def test_no_args(self):
+        """Test case where no args are passed."""
+        result = self.runner.invoke(cli, [])
+        assert result.exception  # Catch any exception
+        assert result.exit_code == 2  # Catch specifically a usage error
```

Comment:
```suggestion
        self.assertEqual(result.exit_code, 2)  # Catch specifically a usage error
```

if you are using the unit test class, do this for other `asserts` as well.